Formula/kns.rb

```diff
@@ -0,0 +1,36 @@
+class Kns < Formula
+  desc "kns - kubernetes namespace switcher"
+  homepage "https://gist.github.com/koenbollen/342c138b6a77d3de6869aad68c193617"
+  url "https://gist.github.com/342c138b6a77d3de6869aad68c193617.git",
+    :revision => "0e5b3848c15b8cbae79c92dcb8f1483edeea6953"
+  head "https://gist.github.com/342c138b6a77d3de6869aad68c193617.git"
+
+  depends_on "fzf"
+
+  def install
+    prefix.install 'kns.bash'
```

Comment:
I think Homebrew Core now supports installing completions:

Not tested, but I think you could add this to the `install` method, and then remove `post_install`:

```
bash_completion.install "kns.bash"
zsh_completion.install "kns.bash"
```